apps/admin-gui/src/app/admin/components/attr-def-list/attr-def-list.component.html

```diff
@@ -56,6 +57,8 @@
   </div>
 </div>
 
+<mat-spinner *ngIf="exporting" class="ml-auto mr-auto"></mat-spinner>
+
```

Comment:
Is it really necessary to have everywhere spinner?

Reply:
While downloading data, third party exporter library I am using here renders all the pages one by one right in the UI, which does not look good. Because of that, I am hiding the table and putting there spinner so the user knows something is going on.